pxr/usd/usdSkel/testenv/testUsdSkelValidators.cpp

```diff
@@ -0,0 +1,88 @@
+//
+// Copyright 2024 Pixar
+//
+// Licensed under the terms set forth in the LICENSE.txt file available at
+// https://openusd.org/license.
+//
+
+#include "pxr/pxr.h"
+#include "pxr/base/tf/token.h"
+#include "pxr/usd/sdr/registry.h"
```

Comment:
Can you remove these extra headers? Thanks

Reply:
updated